samples/subsys/ipc/ipc_service/icmsg/remote/boards/stm32h747i_disco.overlay

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2024 Celina Sophie Kalus <hello@celinakalus.de>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		/delete-property/ zephyr,ipc_shm;
+	};
+
+	/* HSEM IPM is incompatible with HSEM MBOX */
+	/delete-node/ mailbox;
+
+	/* Define new memory regions for TX and RX */
+	/delete-node/ memory@38000000;
+
+	sram_rx: memory@38000000 {
+		reg = <0x38000000 0x08000>;
+	};
+
+	sram_tx: memory@38008000 {
+		reg = <0x38008000 0x08000>;
+	};
+};
+
+&pinctrl {
+	mbox: mbox@58026400 {
+		compatible = "st,mbox-stm32-hsem";
+		reg = < 0x58026400 0x400 >;
+		clocks = < &rcc 0xe0 0x2000000 >;
+		status = "okay";
+		interrupts = < 0x7e 0x0 >;
+		st,hsem_ids_tx = <11>;
+		st,hsem_ids_rx = <10>;
```

Comment:
If we decide to declare list of used channels, I would only list here the ID supported without the RX/TX  property, I look like a duplication of properties defined  in the ipc0 node

what about " st, hsem-channels = <10>, <11>; ?

Reply:
A single hardware semaphore cannot be bidirectional, so the user would have to make sure not to use a single MBOX channel for both sending and receiving. Is that a problem?

Reply:
This is now obsolete, since these properties are removed.